src/style.css

```diff
@@ -0,0 +1,116 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+.card {
+  width: min-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  font-family: "Roboto", sans-serif;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__image {

```

Comment:
image is not centered inside the container
![image](https://user-images.githubusercontent.com/57598825/145799535-64e3b4d3-26eb-4914-a72a-1a5911b48f16.png)